docs/sources/articles/configuring.md

```diff
@@ -96,3 +118,84 @@ These options :
 7. Verify that the `docker` daemon is running as specified wit the `ps` command.
 
 		$ ps aux | grep docker | grep -v grep
+
+### Logs
+
+By default logs for `Upstart` jobs are located in `/var/log/upstart` and the logs for `docker` daemon
+can be located at `/var/log/upstart/docker.log`
+
+    $ tail -f /var/log/upstart/docker.log
+    INFO[0000] Loading containers: done.
+    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
```

Comment:
I am running docker on an Ubuntu 14.04 VM using vagrant with the options `/usr/bin/docker -d --registry-mirror=http://localhost:5000`. I did not specify `aufs` specifically. Can you confirm that `devicemapper` is the default?